optimum/neuron/modeling_diffusion.py

```diff
@@ -78,7 +78,7 @@ def __init__(
         tokenizer: CLIPTokenizer,
         scheduler: Union[DDIMScheduler, PNDMScheduler, LMSDiscreteScheduler],
         feature_extractor: Optional[CLIPFeatureExtractor] = None,
-        device_ids: Optional[List[int]] = [],
+        device_ids: Optional[List[int]] = [0, 1],
```

Comment:
Let's keep it to `None` here and set it to `[0, 1]` if `device_ids=None`. 
It is not good practice to set mutable objects as default values.